templates/account/signup.html

```diff
@@ -4,29 +4,41 @@
 {% trans "Signup" %}
 {% endblock head_title %}
 {% block content %}
-{% element h1 %}
-{% trans "Sign Up" %}
-{% endelement %}
-<p>
-    {% blocktrans %}Already have an account? Then please <a href="{{ login_url }}">sign in</a>.{% endblocktrans %}
-</p>
-{% url 'account_signup' as action_url %}
-{% element form form=form method="post" action=action_url tags="entrance,signup" %}
-{% slot body %}
-{% csrf_token %}
-{% element fields form=form unlabeled=True %}
-{% endelement %}
-{% if redirect_field_value %}
-<input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
-{% endif %}
-{% endslot %}
-{% slot actions %}
-{% element button tags="prominent,signup" type="submit" %}
-{% trans "Sign Up" %}
-{% endelement %}
-{% endslot %}
-{% endelement %}
-{% if SOCIALACCOUNT_ENABLED %}
-{% include "socialaccount/snippets/login.html" with page_layout="entrance" %}
-{% endif %}
+<div class="container mx-auto mt-8">
+    <h1 class="text-3xl mb-6">Create Your Account</h1>
+    <form method="post" action="{% url 'account_signup' %}">
+        {% csrf_token %}
+        <div class="mb-6">
+            <label for="id_email" class="block mb-1 font-medium text-gray-900 dark:text-white">Email Address</label>
+            <input type="email" id="id_email" name="email" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-lime-500 focus:border-lime-500 block w-[50%] p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-lime-500 dark:focus:border-lime-500" placeholder="Email" required />
+            {% for error in email_errors %}
+                <div class="text-sm text-red-800">{{ error }}</div>
+            {% endfor %}
+        </div>
+
+        <div class="mb-6">
+            <label for="id_password1" class="block mb-1 font-medium text-gray-900 dark:text-white">Password:</label>
+            <input type="password" name="password1" class="bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-lime-500 focus:border-lime-500 block w-[50%] p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-lime-500 dark:focus:border-lime-500" placeholder="Password" autocomplete="new-password" required id="id_password1">
+            {% for error in password1_errors %}
+                <div class="text-sm text-red-800">{{ error }}</div>
+            {% endfor %}
+            <ul class="space-y-1 text-gray-500 list-disc list-inside dark:text-gray-400">
+                <li>Your password can't be too similar to your other personal information.</li>
+                <li>Your password must contain at least 8 characters.</li>
+                <li>Your password can't be a commonly used password.</li>
+                <li>Your password can't be entirely numeric.</li>
+            </ul>
```

Comment:
Issue (Non-blocking): This is not needed since you are already outputting the errors on line 23 when the information is incorrect.